docs/data-sources/index.md

```diff
@@ -8,6 +8,56 @@ has_toc: false
 
 # GrimoireLab Projects Definition
 
+Software projects and their related data sources (e.g., Git, GitHub, Slack) are ingested to GrimoireLab via a JSON file generally known as `projects.json`.
+The structure of this file is composed of 3 levels:
+
+1. **Project name**, which includes the name of the project.
+2. **Data sources and metadata.** The former defines the data sources of the project, while the latter allows to include extra information to them.
+3. **Repositories**, which point to the locations of the data. Depending on the data source, they can be URLs (e.g., Git, GitHub) or strings representing for instance Meetup groups, Slack channels or Stackoverflow tags. Furthermore,
+each repository can include additional parameters (identified by `--`) to apply specific filters on the data collect. For instance, the filter `--filter-no-collection=true` allows to ignore
+the repositories that are temporally or permanently unreachable.
+
+The example below summarizes the projects.json structure. As can be seen, `GrimoireLab` and `Excalibur` are the
+projects, which include `git` and  `github` as data sources. The data sources of GrimoireLab are marked as `Stable`, while
+the ones of Excalibur are set as `Alpha`. Finally, the repo `https:/github.com/Bitergia/citadel` includes the param
+`--filter-no-collection=true` which prevents the collection of its data.
```

Comment:
I would like clarification on the "meta" information:

Does "meta" tag data sources or does "meta" specify information about the project?
It looks to me like each project only has one "meta" that will apply to all data sources in that project, thus it seems more appropriate that the meta is describing the project.

Is that so, then we should clarify the language in lines 20-23.

Reply:
I pushed some new changes, but my feeling is that some work is still missing.